src/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics;
 using System.Reflection;
 using System.Runtime.InteropServices;
+using System.Text.Json.Serialization.Converters;
```

Comment:
**Question:** Whenever a nested namespace is referenced within a root namespace like this, it feels odd to me.

Is `DefaultConverters` the right places for the `IsValueConvertable` helper along with the `_valueConverters` dictionary or does it belong somewhere in `JsonClassInfo` maybe or a separate helper class all-together?

Reply:
> it feels odd to me.

It is super odd- and temporary. Figured it was better to stick with odd for now as we rationalize out the extensibility model. (Rather than invent a temporary holder.)